dashboard/test/models/lesson_test.rb

```diff
@@ -935,4 +935,141 @@ def create_swapped_lockable_lesson
     expected_url = "/s/#{script.name}/standards"
     assert_equal expected_url, lesson.course_version_standards_url
   end
+
+  class LessonCopyTests < ActiveSupport::TestCase
+    setup do
+      Script.any_instance.stubs(:write_script_json)
+      Script.stubs(:merge_and_write_i18n)
```

Comment:
are we doing any verification (manual maybe) that scripts.en.yml is getting updated correctly?

Reply:
I'll add some verification that this function is actually called. I manually checked that when copying a lesson, the title is there (instead of the translation missing text)